src/foam/mlang/mlang.js

```diff
@@ -3711,7 +3712,8 @@ foam.CLASS({
     function THEN_BY(a, b) { return this.ThenBy.create({head: a, tail: b}); },
 
     function INSTANCE_OF(cls) { return this.IsInstanceOf.create({ targetClass: cls }); },
-    function CLASS_OF(cls) { return this.IsClassOf.create({ targetClass: cls }); }
+    function CLASS_OF(cls) { return this.IsClassOf.create({ targetClass: cls }); },
+    function CAPABILITY_IS_AVAILABLE(subject) { return this.CapabilityIsAvailable.create({ subject: subject }) }
```

Comment:
mlang is part of FOAM core, but Capabilities are part of NANOS, so some FOAM users may not have capabilities. Better to put this in its own class under crunch.